web/js/libs/browser_utils.js

```diff
@@ -10,6 +10,8 @@
     var hours = now.getHours();
     if (hours >= 12) {
       suffix = ' PM';
+    }
+    if (hours > 12) {
```

Comment:
Have you considered using [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) instead of building the string manually?